src/styles/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
```

create variable for colors
```suggestion
  border: 1px solid $c-someColor;
```